src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -25,6 +25,8 @@ namespace AWS.Deploy.CLI.Commands
 {
     public class DeployCommand
     {
+        private const string REPLACE_TOKEN_STACK_NAME = "{StackName}";

```

Comment:
Why was this changed from `ProjectName` to `StackName`? I assume it is fine but these are the types of changes that would be breaking changes after we have users saving their deployment projects.

Reply:
Updated the PR description to provide clarity.